wd_helper.au3

```diff
@@ -638,40 +638,41 @@ EndFunc   ;==>_WD_FrameLeave
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_FrameList
-; Description ...: get list of documents (window.top and all frames) from current webdriver context/tab
-; Syntax ........: _WD_FrameList($sSession[, $bReturnAsArray = True[, $sFilter = ''[, $bReturnHTML = False]]])
+; Description ...: list detailed information about frames possible to use ("window.top" and all frames)
+; Syntax ........: _WD_FrameList(Const $sSession[, Const $bReturnAsArray = True[, Const $sFilter = ''[, Const $bReturnHTML = False]]])
 ; Parameters ....: $sSession            - Session ID from _WD_CreateSession
 ;                  $bReturnAsArray      - By default on True function will return array, other wise string
-;                  $sFilter             - RegExp pattern to check if desired content exist in frame document HTML source, if used it returns only frames with diesired content
+;                  $sFilter             - RegExp pattern to check HTML from "document.body". If used it returns only frames with desired HTML content
 ;                  $bReturnHTML         - On true function will return frame document HTML source
-; Return values .: Success - array or string
+; Return values .: Success - 2D array (with 6 cols) or string ( delimeted with | and @CRLF )
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
 ;                  - $_WD_ERROR_GeneralError
 ;                  - $_WD_ERROR_Exception
 ;                  - $_WD_ERROR_InvalidExpression
 ;                  - $_WD_ERROR_NotFound
 ; Author ........: mLipok
 ; Modified ......:
-; Remarks .......: using $sFilter will set $bReturnHTML
-; Related .......: __WD_FrameList_Internal, _WD_FrameEnter, _WD_GetFrameCount
+; Remarks .......: using $bReturnHTML will always return array
+; Related .......: _WD_GetFrameCount, _WD_FrameEnter, _WD_FrameLeave
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func _WD_FrameList($sSession, $bReturnAsArray = True, $sFilter = '', $bReturnHTML = False)
+Func _WD_FrameList(Const $sSession, Const $bReturnAsArray = True, Const $sFilter = '', Const $bReturnHTML = False)
```

Comment:
Using `Const` on a function's parameters isn't commonly done, even when you don't intend to modify the parameter's value. So why do it here?

Reply:
I tried whether it would work, and I wanted to know your opinion on this.

Ok.
Soon I revert the `const` change from `_WD_FrameLinst` parameters.

Reply:
Please take a look here: [const parameters + header](https://github.com/Danp2/au3WebDriver/pull/362/commits/6276ff5011d2a1cc604dbffceb19079f7100ac47)